src/pages/billing/Project.js

```diff
@@ -233,6 +262,23 @@ const ProjectDetail = ({ authorizeAndLoadAccounts, billingAccounts, billingProje
       const spend = await Ajax(signal).Billing.getSpendReport({ billingProjectName: billingProject.projectName, startDate, endDate })
       const costFormatter = new Intl.NumberFormat(navigator.language, { style: 'currency', currency: spend.spendSummary.currency })
       setTotalCost(costFormatter.format(spend.spendSummary.cost))
+
+      const workspaceDetails = _.find(details => details.aggregationKey === 'Workspace')(spend.spendDetails)
+      console.assert(workspaceDetails !== undefined, 'Spend report details do not include aggregation by Workspace')
```

Comment:
If this assertion fails, we will not render the chart, but the user will see no error (that's the magic of `workspaceDetails?.spendData`). The error will appear in the JavaScript console.

Reply:
I got the impression from your PR description that this could be fairly common, thus noisy. Would it be better to display this information in place of the chart so that if I were expecting to see spending details I would have some indication of why they are not there?

Reply:
No, this is not common. This would only happen if the API response was not formatted as we expected.

In the case of billing data not existing at all, we wouldn't be getting to this point.